### PR TITLE
[ADD] fieldservice_googlemaps

### PR DESCRIPTION
**Pull request by osimallen**
Created a new module to add address auto-completion and geometry sync between orders and associated locations.

### CODE REVIEW

File: fieldservice-googlemaps/__init__.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2018 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import models
```
Comment (max3903) [inline fieldservice-googlemaps/__init__.py]: Missing newline

File: fieldservice-googlemaps/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Copyright (C) 2018 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from . import fsm_location, fsm_order
```
Comment (max3903) [inline fieldservice-googlemaps/models/__init__.py]: Missing newline

File: fieldservice-googlemaps/models/fsm_location.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+    @api.multi
+    def write(self,vals):
+        res = super(FSMLocation,self).write(vals)
+        if ('partner_latitude' in vals) and ('partner_longitude' in vals):
+            lat = vals['partner_latitude']
+            lng = vals['partner_longitude']
+            pnt = geo_fields.GeoPoint.from_latlon(cr=self.env.cr,
+                                                latitude=lat,
+                                                longitude=lng)
```
Comment (max3903) [inline fieldservice-googlemaps/models/fsm_location.py]: Wrong indentation

File: fieldservice-googlemaps/models/fsm_order.py [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        if lat and lng:
+            pnt = geo_fields.GeoPoint.from_latlon(cr=self.env.cr,
+                                                      latitude=lat,
+                                                      longitude=lng)
+            self.shape = pnt
+
+
+
+
+
```
Comment (max3903) [inline fieldservice-googlemaps/models/fsm_order.py]: Empty lines to remove

File: fieldservice-googlemaps/views/fsm_location.xml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+                <attribute name='invisible'>1</attribute>
+            </field>
+            <field name='partner_longitude' position='attributes'>
+                <attribute name='invisible'>1</attribute>
+            </field>
+        </field>
+    </record>
+
+    
+
```
Comment (max3903) [inline fieldservice-googlemaps/views/fsm_location.xml]: Empty lines to remove

**Review by max3903 — CHANGES_REQUESTED** [review]
Can you rename the module to fieldservice_googlemaps with underscore?

File: fieldservice-googlemaps/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+{
+    "name": "Field Service Google Map Integration",
+    "version": "11.0.1.0.0",
+    "license": "AGPL-3",
+    "summary": "",
+    "author": "Open Source Integrators",
+    "maintainer": "Open Source Integrators",
+    "website": "https://www.opensourceintegrators.com",
+    "depends": [
+        'web_google_maps',
```
Comment (max3903) [inline fieldservice-googlemaps/__manifest__.py]: `'fieldservice',`?

File: fieldservice-googlemaps/__init__.py (hunk 1 above)
Comment (osimallen) [reply fieldservice-googlemaps/__init__.py]: Resolved in latest commit

File: fieldservice-googlemaps/models/__init__.py (hunk 2 above)
Comment (osimallen) [reply fieldservice-googlemaps/models/__init__.py]: Resolved in latest commit

File: fieldservice-googlemaps/models/fsm_location.py (hunk 3 above)
Comment (osimallen) [reply fieldservice-googlemaps/models/fsm_location.py]: Resolved in latest commit

File: fieldservice-googlemaps/models/fsm_order.py (hunk 4 above)
Comment (osimallen) [reply fieldservice-googlemaps/models/fsm_order.py]: Resolved in latest commit

File: fieldservice-googlemaps/views/fsm_location.xml (hunk 5 above)
Comment (osimallen) [reply fieldservice-googlemaps/views/fsm_location.xml]: Resolved in latest commit

File: fieldservice-googlemaps/__manifest__.py (hunk 6 above)
Comment (osimallen) [reply fieldservice-googlemaps/__manifest__.py]: Resolved in latest commit

**Review by max3903 — APPROVED** [review]

**Pull request merged**